documentation/docs/04-advanced-routing.md

```diff
@@ -144,6 +144,8 @@ Some parts of your app might need something other than the default layout. For t
 <h1>I am inside +layout-foo</h1>
 ```
 
+> Named layout should only be referenced from `+page.svelte` files
```

Comment:
Just realized this isn't enough, it also applies to (named) layout files. I prefer to not be too wordy, here so I'll change this to
```suggestion
> Named layout should only be referenced from Svelte files
```